docs/assets/ie10-viewport-bug-workaround.js

```diff
@@ -0,0 +1,24 @@
+/*!
+ * IE10 viewport hack for Surface/desktop Windows 8 bug
+ * Copyright 2014-2017 The Bootstrap Authors
+ * Copyright 2014-2017 Twitter, Inc.
+ * Licensed under MIT (https://github.com/twbs/bootstrap/blob/master/LICENSE)
+ */
+
+// See the Getting Started docs for more information:
+// https://getbootstrap.com/getting-started/#support-ie10-width
+
+(function () {
```

Comment:
Add comments on what behaviour we are mitigating

Reply:
It is a fix for viewport issues faced in IE10 when using Twitter Bootstrap. It is a vendor based file.